number_of_simulated_objects: 20
simulated_object_name: ball
simualtion_sensor_object_threshold: 0.5
